beginend.el

```diff
@@ -239,6 +239,13 @@ BEGIN-BODY and END-BODY are two `progn' expressions passed to respectively
   (progn
     (prodigy-last)))
 
+(beginend-define-mode magit-status-mode
+  (progn
+    (re-search-forward "^$")
+    (forward-line))
+  (progn
+    (beginend--goto-nonwhitespace)))
```

Comment:
thank you very much. I implemented your suggestion. I guess that for magit-status, imenu support makes more sense than beginend. But it's still cool to have it working. Thanks.